pkg/skaffold/deploy/docker/deploy.go

```diff
@@ -186,6 +260,12 @@ func (d *Deployer) Cleanup(ctx context.Context, out io.Writer) error {
 		d.portManager.relinquishPorts(container.Name)
 	}
 
+	for _, m := range d.debugger.SupportMounts() {
+		if err := d.client.VolumeRemove(ctx, m.Source); err != nil {
+			return errors.Wrap(err, "cleaning up debug support volume")
+		}
+	}
```

Comment:
Do we need to clear up `d.debugger.supportMounts`?  What if another container is referencing the volume?

Reply:
these support mounts have all been created in the context of this specific skaffold run, so no other containers can know about them except the ones created by skaffold. we shouldn't need to do anything with these, since they're only referenced in-memory.

Reply:
Oh right: the volumes are created with a unique name, so they won't be shared with other instances.